src/limbo/init/random_sampling.hpp

```diff
@@ -20,7 +18,6 @@ namespace limbo {
                     Eigen::VectorXd new_sample(F::dim_in);
                     for (int i = 0; i < F::dim_in; i++)
                         new_sample[i] = tools::rand<double>(0, 1);
-                    std::cout << "random sample:" << new_sample.transpose() << std::endl;
```

Comment:
Because in no other initialization function we output anything. We shouldn't be inconsistent. It shouldn't really be part of this PR, but I just saw it and did it.